viewer/packages/data-providers/src/pointcloud-stylable-object-providers/cdfAnnotationsToObjects.ts

```diff
@@ -26,7 +26,8 @@ function cdfAnnotationsToPointCloudObjects(cdfAnnotations: CdfPointCloudObjectAn
 
     return {
       annotationId: cdfAnnotation.annotationId,
-      assetId: cdfAnnotation.assetId,
+      assetId: cdfAnnotation.asset?.id,
+      assetRef: cdfAnnotation.asset,
```

Comment:
Strictly speaking, not necessary - the reason it's here is that `PointCloudObjectMetadata` has an `assetId` field from before which is publicly exposed, which means we can't remove it. It's marked as deprecated, but that's all we can do. In this case in particular, the `assetId` field won't be used at all, because the object returned to the user will be reconstructed, and will use the id in the `assetRef` anyway. So this is just for consistency and avoiding confusing errors in the future.